operations/invoices.md

```diff
@@ -4,7 +4,11 @@
 ## Get invoices
 
 This operation returns a list of invoices.
-Note: This operation needs [Authentication](../essential-guide/authentication.md) and supports the `include` query parameters `invoice_item, user, register, product_variant, invoice_item_modifier` following the [JSON:API specification](../essential-guide/features.md).
+Note: This operation needs [Authentication](../essential-guide/authentication.md) and supports following the [JSON:API features](../essential-guide/features.md):
+
+- [Relationships](../essential-guide/relationships.md); `invoiceItem`, `user`, `register`, `productVariant`, `invoiceItemModifier` using `include` query parameter.
```

Comment:
@MikeAdamsMews I think we had some typos in the content. All `to-on` relationships should be singular and all `to-many` be plural. In this case, this should be plural because many `invoiceItems` can belong to an invoice.